tracker/tracker_debugger_api.yaml

```
@@ -1318,3 +1374,912 @@ components:
+          example: '2020-02-01T00:00:00.000000000Z'
+        ip:
+          description: 'Tracker client IP (real value can be provided in headers or `cip` parameter)'
+          type: string
+          format: ip
+          example: 75.13.13.87
+        host:
+          description: 'A host under which app is available'
+          type: string
+          example: piwikpro.tld
+        scheme:
+          description: 'Scheme used for the request'
+          type: string
+          example: http
+        method:
+          description: 'Method used for the request'
+          type: string
+          example: GET
+        headers:
+          description: Request HTTP headers
+          type: array
+          items:
+            type: array
+            items:
+              type: string
+          example:
+            - - 'x-forwarded-for'
+              - '172.18.0.15'
+            - - 'accept-language'
+              - 'en-US,en;q=0.9'
+            - - 'user-agent'
+              - 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.117 Safari/537.36'
+        query_params:
+          description: 'Tracking request parameters'
+          type: object
+          $ref: '#/components/schemas/TrackingParameters'
+        cookie_id:
+          description: 'Cookie ID sent in the _id parameter'
+          type: string
+          example: eddf6b9aa9d8fb32
+        user_id:
+          description: 'User ID sent in the uid parameter'
+          type: string
+          example: 'particular-user_login'
+        body:
+          description: 'Request body payload'
+          type: string
+          example: "idsite=1&rec=0&action_name=This%20won%27t%20be%20recorded"
+        debug:
+          description: 'States whether request is used for debugging'
+          type: boolean
+          default: false
+          example: true
+    TrackingParameters:
+      description: A tracking log
+      required:
+        - idsite
+      type: object
+      properties:
+        idsite:
+          description: ID of the tracked app/website
+          type: string
+          example: 4
+        _idts:
+          description: 'Time of first visit'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 1577836800
+        _viewts:
+          description: 'Time of previous visit'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 1580342400
+        _idvc:
+          description: 'Visit counter'
+          type: integer
+          minimum: 0
+          maximum: 4294967295
+          example: 2
+        url:
+          description: 'URL of the action'
+          type: string
+          example: 'https://piwik.pro/'
+        urlref:
+          description: 'HTTP referrer'
+          type: string
+          example: 'https://clearcode.cc/'
+        res:
+          description: 'Resolution of the visitor''s device in pixels'
```

Use double quotes when string has single quotes.
```suggestion
          description: "Resolution of the visitor's device in pixels"
```

---

Why?

---

I've escaped it the way documentation suggests. Even Swagger editor prefers single quotes.